### PR TITLE
2.la terminal

### PR DESCRIPTION
**Pull request by NahuelFerreroG**
# 🤘 Courses PRs

## Type

- [x] Translation
- [x] Enhacement

### CODE REVIEW

**Comment by NahuelFerreroG** [comment]
@maurodibert in line 38 add a small clarification that in windows it is not like this

File: dart/11.Medium_Dart/2_The_terminal.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Did you research? What the `-R` does is to make the deletion recursive since it is a directory that may contain more files or directories.
+
+On the other hand, you will see when you start doing some research, the words `su` and `sudo`. I didn't tell you before, but __every operating system works with different users with different roles__, which allows us to prevent that __someone__ who shouldn't __do something he/she shouldn't__.
+
+__The most powerful user is called `root`__ and is the one who has all the permissions. If you type `su`, you will be asked for the _password_ to log in as that user. On Mac for example, this user is disabled by default. On the other hand, if you type `sudo` (_Super User DO_), this command gives the current user administrator permissions. Be careful! There is no `sudo` command in Windows, the nearest equivalent is `runas`(_run as administrator_).
```
Comment (NahuelFerreroG) [inline dart/11.Medium_Dart/2_The_terminal.md:38]: add a small clarification that in windows it is not like this

File: dart_sp/11.Dart_medio/2_La_terminal.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Buscaron!? Lo que hace la `-R` es hacer recursiva la eliminación ya se trata de
 
 Por otro lado, van a ver cuando empiecen a investigar un poco, las palabras 'su' y 'sudo'. No se los dije antes, pero __cada sistema operativo funciona con distintos usuarios los cuales cuentan cada uno con diferentes roles__, lo que nos permite evitar que __alguien__ que no debe __haga algo que no debe__.
 
-__El usuario más poderoso se llama `root`__ y es quien tiene todos los permisos. Si ustedes teclean `su`, les va a pedir el _password_ para loguearse como dicho usuario. En Mac por ejemplo, dicho usuario está desactivado por defecto. Por otro lado, si uno teclea `sudo` (_Super User DO_), dota al usuario actual de permisos de administrador.
+__El usuario más poderoso se llama `root`__ y es quien tiene todos los permisos. Si ustedes teclean `su`, les va a pedir el _password_ para loguearse como dicho usuario. En Mac por ejemplo, dicho usuario está desactivado por defecto. Por otro lado, si uno teclea `sudo` (_Super User DO_), dota al usuario actual de permisos de administrador. Cuidado! no existe el comando `su`o `sudo`en windows, el mas parecido es `runas`.
```
Comment (NahuelFerreroG) [inline dart_sp/11.Dart_medio/2_La_terminal.md:38]: add a small clarification that in windows it is not like this

Comment (mal2tin) [reply dart_sp/11.Dart_medio/2_La_terminal.md:38]: > 

I have never used windows for programming... but if "runas" is mentioned, I think it is also necessary to clarify that it is not needed in this specific case. I mean "el más parecido es "runas" (aunque no aplica en este caso)." On the other hand, "más" takes tílde.

Comment (NahuelFerreroG) [reply dart_sp/11.Dart_medio/2_La_terminal.md:38]: I understand and agree, the thing is that on the other hand I've never used linux so I do not know if this was needed xD. It was just a comment cause when I was translating I was also trying things and su or sudo never work for me and it was exhausting. The same hapened whith other commands.

maybe something erase that, and at the end after all the other commnads add a comment that many of those commnads do not work with windows.

I`ll make another commit

File: dart_sp/11.Dart_medio/2_La_terminal.md [hunk 3]
```diff
@@ -44,9 +44,9 @@ Como para terminar esta capaz no tan breve introducción, les dejo un listado de
...
-- `mv <opciones> <archivo> <ruta>`: mueve un archivo o directorio a un archivo o directorio. Se utiliza también para renombrar; el origen y destino es un archivo.
+- `mv <opciones> <archivo> <ruta>`: se utiliza  para mover archivos y directorios de un directorio a otro, o para renombrar un archivo o directorio.
 - `cat <opciones> <archivos>`: muestra por pantalla el contenido de los archivos por la salida estándar.
 -`find <ruta(s)> <condición(es) de búsqueda> <acciones>`: busca archivos recorriendo árboles de directorios especificados por las rutas, evaluando de izquierda a derecha las condiciones de búsqueda. Además se pueden especificar acciones sobre los resultados obtenidos.
 -`pwd <opciones>`: Muestra el nombre del directorio en el que uno se encuentra situado (_print working directory_).
 
-Espero que tengan suficiente para meter sus manitas a explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️
+Podrán ver que muchos de estos comandos no funcionan en Windows, pero que eso no los detenga de explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️
```
Comment (maurodibert) [inline dart_sp/11.Dart_medio/2_La_terminal.md:52]: Si te parece, yo buscaría ya que los estás conociendo mientras los hacés, cuáles son los que faltan e ir sumándolos gradual y puntualmente.

**Review by maurodibert — CHANGES_REQUESTED** [review]

File: dart_sp/11.Dart_medio/2_La_terminal.md [hunk 4]
```diff
@@ -44,9 +44,9 @@ Como para terminar esta capaz no tan breve introducción, les dejo un listado de
 - `kill PID`: mata un programa utilizando su PID. Por ejemplo: `kill 4320`.
 - `ls <opciones> <ficheros>`: lista el contenido de un directorio.
 - `cp <opciones> <archivo> <ruta>`: copia un archivo o directorio en el directorio especificado.
-- `mv <opciones> <archivo> <ruta>`: mueve un archivo o directorio a un archivo o directorio. Se utiliza también para renombrar; el origen y destino es un archivo.
+- `mv <opciones> <archivo> <ruta>`: se utiliza  para mover archivos y directorios de un directorio a otro, o para renombrar un archivo o directorio.
```
[the remark this answers is not in the document]
Comment (NahuelFerreroG) [reply dart_sp/11.Dart_medio/2_La_terminal.md:47]: done!

File: dart_sp/11.Dart_medio/2_La_terminal.md [hunk 5]
```diff
@@ -18,11 +18,11 @@ __Lo primero que van a ver, es el _prompt_ o prontuario__ en español y es la ay
...
 
-__Las opciones son diferentes posibilidades de usar el comando y las podemos ingresar utilizando `--` antes de la opción, o un solo `-` antes de la abreviatura__ de la opción, que puede ser la primer letra o las primeras dos. Por ejemplo tenemos el comando `dart`, que vendría ser una utilidad en la terminal para desarrollar en Dart. Si solo tecleamos `dart`, esto es lo que nos aparece:
+__Las opciones son diferentes posibilidades de como usar el comando y las podemos ingresar utilizando `--` antes de la opción, o un solo `-` antes de la abreviatura__ de la opción, que puede ser la primer letra o las primeras dos. Por ejemplo tenemos el comando `dart`, que vendría ser una utilidad en la terminal para desarrollar en Dart. Si solo tecleamos `dart`, esto es lo que nos aparece:
```
[the remark this answers is not in the document]
Comment (NahuelFerreroG) [reply dart_sp/11.Dart_medio/2_La_terminal.md:21]: done!

File: dart_sp/11.Dart_medio/2_La_terminal.md (hunk 3 above)
Comment (NahuelFerreroG) [reply dart_sp/11.Dart_medio/2_La_terminal.md:52]: Seems good, but as a future task. I don't think it's really necessary for the course or for me right now. If you agree, we leave this for later.
Maybe one more modification: from this "Podrán ver que muchos de estos comandos no funcionan en Windows, pero que eso no los detenga de explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️"
to this "Si estan trabajando en Windows podrán ver que muchos de estos comandos no funcionan (algunos tienen otro nombre), pero que eso no los detenga de explorar, probar y hacerlo sobre todo frente a personas innotas para hacerse ver como superhéroes del teclado! 🦸‍♂️"

**Review by maurodibert — APPROVED** [review]

**Pull request merged**